kafka-versions.yaml

```diff
@@ -35,11 +35,19 @@
   third-party-libs: 2.3.x
   default: false
   unsupported-features: kafkaMirrorMaker2
-- version: 2.4.0
+# - version: 2.4.0
```

Comment:
Given how the upgrade works, we need to at least for one release support both 2.4.0 and 2.4.1 ... so I think this section needs to be uncommented.